.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+Thank you for contributing to jenkins-infra/charts. Before you submit this PR we'd like to
+make sure you are aware of our technical requirements and best practices:
+
+* https://github.com/jenkins-infra/charts/blob/master/CONTRIBUTING.md#technical-requirements
+* https://github.com/helm/helm-www/tree/master/content/en/docs/chart_best_practices
+
+For a quick overview across what we will look at reviewing your PR, please read
+our review guidelines:
+
+* https://github.com/jenkins-infra/charts/blob/master/REVIEW_GUIDELINES.md
+
+Following our best practices right from the start will accelerate the review process and
+help get your PR merged quicker.
+
+When updates to your PR are requested, please add new commits and do not squash the
+history. This will make it easier to identify new changes. The PR will be squashed
+anyways when it is merged. Thanks.
+
+For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.
+
+Please make sure you test your changes before you push them. Once pushed, a Jenkins job
+will run across your changes and do some initial checks and linting. These checks run
+very quickly. Please check the results. We would like these checks to pass before we
+even continue reviewing your changes.
+-->
+
+#### Is this a new chart
+- [ ] `CODEOWNERS` is updated
+- [ ] If secrets are needed, `.sops.yaml` must be updated with appropriated public gpg key ID, and full public gpg key must be provided to @olblak
+
+#### What this PR does / why we need it:
+
+#### Which issue this PR fixes
+
+  - fixes # [INFRA-XXXX](https://issues.jenkins-ci.org/projects/INFRA/issues/INFRA-XXXX)
+
+#### Special notes for your reviewer:
+
+#### Checklist
+[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
+- [ ] [CLA](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
```

Comment:
what benefit does this get us? it's just a pain and I wouldn't expect humans to check this?

Reply:
> what benefit does this get us? it's just a pain and I wouldn't expect humans to check this?

 While I agree with you, I am hesitating between one of the following options:
- We just encourage people to sign the Jenkins CLA
- We make it mandatory for everybody
- We make it mandatory for people who merge the code (if it really make sense)
- We don't care
- We use the same approach than Helm project [link](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work)

Reply:
that doesn't answer for what benefit we get?

Given the very low number of contributions we get here, is it worth adding barriers?

Reply:
I find checklists in pull request templates quickly go from cool to never filled out.

I'm cool with the cla requirement but can we use a bot instead of a manual process? I will volunteer to modify the probot to check the cla repo

Reply:
I removed the CLA sections for now, I need more time to study that area and see how we can improve the jenkins CLA